src/ansys/fluent/core/solver/flobject.py

```diff
@@ -1749,8 +1751,22 @@ def __len__(self):
 
 
 class CreatableNamedObjectMixin(collections.abc.MutableMapping, Generic[ChildTypeT]):
-    """Provides creatable named objects."""
+    """Provides creatable named objects for Fluent 2025 R1 and later."""
 
+    def __setitem__(self, name: str, value):
+        if name not in self.get_object_names():
+            with self._while_creating():
+                self.flproxy.create(self.path, name)
+        child = self._objects.get(name)
+        if not child:
+            child = self._create_child_object(name)
+        child.set_state(value)
+
+
+class CreatableNamedObjectMixinOld(CreatableNamedObjectMixin):
+    """Provides creatable named objects for Fluent 24.2 and earlier."""
+
+    # In Fluent 25.1, create method is available as commands in NamedObject class.
```

Comment:
```suggestion
    """Provides creatable named objects for Fluent 2024 R2 and earlier."""

    # In Fluent 2025 R1, the ``create()`` method is available as commands in the ``NamedObject`` class.
```
User-facing doc should format Ansys versions like this: 2024 R2. In the comment, the edits to other than the release use the double backticks to surround code entites.

Reply:
Actually, this is a code comment. It won't be visible in the user-facing doc.

Reply:
True, but the previous class description will be. It is best to keep the terminology consistent!